### DISCUSSION TITLE
Added addtional Ender 3 V2 Profiles.

### DISCUSSION THREAD

**Discussion by Littleaton** [OrcaSlicer/OrcaSlicer#4192] [Ideas]
For those that need these, here are additional nozzle profiles for the ender 3 v2.

Just paste these into the install folder: 
C:\Program Files\OrcaSlicer\resources\profiles\Creality\machine

[Creality Ender-3 V2 0.6 nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/14378200/Creality.Ender-3.V2.0.6.nozzle.json)
[Creality Ender-3 V2 0.8 nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/14378201/Creality.Ender-3.V2.0.8.nozzle.json)
[Creality Ender-3 V2.json](https://github.com/SoftFever/OrcaSlicer/files/14378202/Creality.Ender-3.V2.json)
[Creality Ender-3 V2 0.2 nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/14378203/Creality.Ender-3.V2.0.2.nozzle.json)
[Creality Ender-3 V2 0.4 nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/14378204/Creality.Ender-3.V2.0.4.nozzle.json)

the same can be done for any other machine, just need to create the files and edit them to the nozzle size.

**Comment by ecomcodeLab** [comment]
we have to add them in resources\profiles\Creality\Creality.json

like this:
```
        {
            "name": "Creality Ender-3 V2 0.8 nozzle",
            "sub_path": "machine/Creality Ender-3 V2 0.8 nozzle.json"
        },
```

i dont know if the settings will be deleted if we make a update. 
        
but if so i think the profiles still there so we can use them otherwise

@Littleaton the 0.4 and Creality.Ender-3.V2.json are the default ones right ????

**Reply by Littleaton** [reply]
Ah lets see, I posted that ages ago.  But no, at the time there was not a v2 option or at least I could not find one, so made one.

**Reply by ecomcodeLab** [reply]
yea please look in the files i have added to clearity.json them and used your files .but they are still not shown [Creality Ender-3 V2.json](https://github.com/SoftFever/OrcaSlicer/files/14378202/Creality.Ender-3.V2.json) this is your file i replaced my one with this

**Reply by ecomcodeLab** [reply]
when i add a new one and then use the option creat a printer from template i can see the new 0.2 nozzle .... but they want show in the settings for a template printer. 

maybe there are some other files that we have to modify

**Reply by Littleaton** [reply]
the correct file is "C:\Program Files\OrcaSlicer\resources\profiles\Creality.json"
try adding to that.

**Reply by ecomcodeLab** [reply]
i added but nothing shown under
```

    "machine_list": [
        {
            "name": "Creality Ender-3 V2 0.4 nozzle",
            "sub_path": "machine/Creality Ender-3 V2 0.4 nozzle.json"
        },
        {
            "name": "Creality Ender-3 V2 0.2 nozzle",
            "sub_path": "machine/Creality Ender-3 V2 0.2 nozzle.json"
        },
        {
            "name": "Creality Ender-3 V2 0.6 nozzle",
            "sub_path": "machine/Creality Ender-3 V2 0.6 nozzle.json"
        },
        {
            "name": "Creality Ender-3 V2 0.8 nozzle",
            "sub_path": "machine/Creality Ender-3 V2 0.8 nozzle.json"
        },
```

and yes files exist there i have checked the file path.  when you go to system presets they want show

when we try to go to "create Printer" then type:  create printer  some settings go to next page there "create from template" the new templates are shown but when i click create from preset it wirtes: preset is not found

**Reply by ecomcodeLab** [reply]
maybe becorse they have no filament and processor presets that can be the problem

**Reply by Littleaton** [reply]
At this point my suggestion would be to just edit any one of the templates there in orca itself and make it what ever your printer is by setting the bed size/etc.

Takes longer to figure this out then just creating your printer in there.
